examples/regression/4-tutorials/plot_ts-tutorial.py

```diff
@@ -0,0 +1,337 @@
+"""
```

Comment:
In the examples, the time series tutorial shows up first, any way to make the tabular regression show up first?

Reply:
I'm not sure how to do it but this is not important.